components/tidb_query_expr/src/impl_json.rs

```diff
@@ -156,6 +156,47 @@ pub fn json_merge(args: &[Option<JsonRef>]) -> Result<Option<Json>> {
     Ok(Some(Json::merge(jsons)?))
 }
 
+#[rpn_fn(nullable, raw_varg)]
+#[inline]
+fn json_quote(arg: &[ScalarValueRef]) -> Result<Option<Bytes>> {
+    quote(arg[0].as_bytes())
+}
+
+fn quote(input: Option<BytesRef>) -> Result<Option<Bytes>> {
```

Comment:
Is this implementation the same as `strconv.Quote` in Golang?

ref: https://github.com/pingcap/tidb/blob/master/expression/builtin_json.go#L1122

Reply:
It that ok to import serde here? I think this should be more reliable.

Reply:
Why we need to import `serde` here?

Reply:
Use `serde_json::to_string()` to replace this  `quote`.

Reply:
Are they consistent with the TiDB side of the implementation?